OpenRA.Mods.Common/Scripting/Global/WidgetGlobal.cs

```diff
@@ -0,0 +1,35 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Drawing;
+using OpenRA.Graphics;
+using OpenRA.Mods.Common.Widgets;
+using OpenRA.Scripting;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Scripting.Global
+{
+	[ScriptGlobal("MissionUI")]
```

Comment:
When your merge-able chrome yaml comes into play this can be used in a lot more flexible ways. Already this isn't tied to narrative campaign missions, but also useful for mini games.